framework/docker/cosmos/hyperlane_provider.go

```diff
@@ -0,0 +1,91 @@
+package cosmos
+
+import (
+	"context"
+	"fmt"
+	"github.com/celestiaorg/tastora/framework/docker/hyperlane"
+)
+
+var _ hyperlane.ChainConfigProvider = (*Chain)(nil)
+
+// GetHyperlaneRegistryEntry returns the registry entry (metadata + addresses) for this chain.
+func (c *Chain) GetHyperlaneRegistryEntry(ctx context.Context) (hyperlane.RegistryEntry, error) {
+	networkInfo, err := c.GetNetworkInfo(ctx)
+	if err != nil {
+		return hyperlane.RegistryEntry{}, err
+	}
+
+	meta := hyperlane.ChainMetadata{
+		ChainID:     c.GetChainID(),
+		DomainID:    69420,
```

Comment:
Maybe we can spin this out into a const somewhere? Like `TestDomainID` or something?
Doesn't necessarily have to be this number, can be whatever for e2e tests